JapaneseFilter/sections/specific.txt

```diff
@@ -3693,6 +3693,9 @@ bbs4.mozakin.com###trend_pain
 bbs4.mozakin.com##.trend_body > div:first-child
 bbs4.mozakin.com##.trend_body div[style^="background-color: black; overflow: hidden; color: white"]
 nagasaki-np.co.jp,bbs4.mozakin.com##.txtad
+nagasaki-np.co.jp##.div_sns_share
```

Comment:
It seems that it still not moved to Social Media filter - https://github.com/AdguardTeam/AdguardFilters/pull/177751#discussion_r1577463102
Please move it somewhere here - https://github.com/AdguardTeam/AdguardFilters/blob/master/SocialFilter/sections/specific.txt#L77-L78